src/style.css

```diff
@@ -1,3 +1,76 @@
-body {
+:root {
+  --blue: #00acdc;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  padding: 0 50px;
+  margin: 0;
+  justify-content: flex-end;
+}
+
+.nav {
+  display: flex;
+  justify-content: flex-end;
+  padding-left: 0;
+}
+
+.header .nav__item {
+  display: flex;
+  margin-left: 20px;
+  height: 60px;
+  line-height: 60px;
+  text-align: center;
+  padding: 0;
+}
+
+.nav__link {
+  text-align: center;
+  text-transform: uppercase;
+  color: #000;
+  text-decoration: none;
+}
+
+.nav__list {
+  justify-content: flex-end;
+  display: flex;
   margin: 0;
+  padding-left: 0;
+}
+
+.moyo_img {
+  position: absolute;
+  left: 50px;
+  top: 10px;
+}
+
+.is-active {
+  position: relative;
+  color: var(--blue);
+}
+
+.is-active::after {
+  content: '';
+  height: 4px;
+  background-color: var(--blue);
+  border-radius: 8px;
+  width: 100%;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+}
+
+.nav__link:hover {
+  color: var(--blue);
+}
+
+.header .nav__item:first-child {
```

Comment:
```suggestion
.nav__item:first-child {
```

Reply:
it's better to write styles like groups, it should be under .nav__item styles

Now you have everything mixed and it's hard to read